api/src/main/java/io/strimzi/api/kafka/model/listener/arraylistener/GenericKafkaListenerConfiguration.java

```diff
@@ -70,17 +70,19 @@ public void setBrokerCertChainAndKey(CertAndKeySecretSource brokerCertChainAndKe
         this.brokerCertChainAndKey = brokerCertChainAndKey;
     }
 
-    @Description("Configures the `Ingress` class that defines which `Ingress` controller will be used. " +
-            "This field can be used only with `ingress` type listener. " +
-            "If not specified, the default Ingress controller will be used.")
+    @Description("Configures specific class for `Ingress` and `LoadBalancer` that defines which controller will be used. " +
+            "This field can be used only with `ingress` and `loadbalancer` type listener. " +
+            "If not specified, the default controller will be used." +
+            "For Ingress listener the service spec property `ingressClassName` will be set" +
+            "For LoadBalancer listener the service spec property `loadbalancerClass` will be set")
```

Comment:
The leading spaces are missing + some possible improvements.

```suggestion
            "If not specified, the default controller will be used. " +
            "For Ingress listener the property `ingressClassName` will be set in the Ingress resources. " +
            "For LoadBalancer listener the property `loadBalancerClass` will be set in the Service resources.")
```